packages/components/src/tools-panel/test/__snapshots__/index.js.snap

```diff
@@ -89,15 +89,15 @@ exports[`ToolsPanel first and last panel items should apply first/last classes t
   max-width: 100%;
 }
 
-.emotion-6.emotion-6 .e1puf3u4 {
+.emotion-6.emotion-6 .ej5x27r4 {
```

Comment:
These hashes changed because the extension of the styles file changed from `.js` to `.ts` 😮

Reply:
A-ha! At least we confirmed the reason why they changed